### ISSUE TITLE
Allow to set up multiple DBs per environment variables

### ISSUE THREAD

**Issue by marton78** [blacklabelops/postgres#3]
Original issue by @evleite from [here](https://github.com/blacklabelops/jira/issues/13): 

It would be great to configure more than one db on the same container and it could be used on docker compose file like this:

docker run --name postgres -d 
-e 'POSTGRES_USER=jira' 
-e 'POSTGRES_PASSWORD=jellyfish' 
-e 'POSTGRES_DB1=jiradb' 
-e 'POSTGRES_ENCODING1=UNICODE' 
-e 'POSTGRES_COLLATE1=C' 
-e 'POSTGRES_COLLATE_TYPE1=C' 
-e 'POSTGRES_DB2=confluencedb' 
-e 'POSTGRES_ENCODING2=UNICODE' 
-e 'POSTGRES_COLLATE2=C' 
-e 'POSTGRES_COLLATE_TYPE2=C' 
-e 'POSTGRES_DB3=bitbucketdb' 
-e 'POSTGRES_ENCODING3=UNICODE' 
-e 'POSTGRES_COLLATE3=C' 
-e 'POSTGRES_COLLATE_TYPE3=C' 
blacklabelops/postgres

This way we can have one single db server/container.

**Comment by blacklabelops** [comment]
I have closed that issue because it belongs on my postgres repository, not on my jira repository. The postgres image rework is still in the making.

**Comment by marton78** [comment]
Yeah I know, you asked the original author to file an issue here, but he seems to have forgotten. So I did it.

**Comment by blacklabelops** [comment]
Yeah now I see. Absolutely correct and many thanks!
